providers/ruleset.rb

```diff
@@ -18,9 +18,12 @@
 #
 
 # provider for installing audit rules from a template
+
 action :create do
-  template '/etc/audit/audit.rules' do
+  t = template '/etc/audit/audit.rules' do
     source "#{new_resource.name}.erb"
-    notifies :restart, resources(service: 'auditd')
+    notifies :run, 'execute[auditctl -R]', :immediately
```

Comment:
Please substantiate your claims.

The init.d script is still used to start auditd and so `/etc/default/auditd` is still used at least for `EXTRAOPTIONS` and `USE_AUGENRULES`, though the latter isn't really compatible with the cookbook anyway.

The cookbook will never and should never attempt to stop auditd so `AUDITD_CLEAN_STOP` and `AUDITD_STOP_DISABLE` will not be used in this context. That doesn't prevent them from being used on shutdown or if auditd is stopped manually.